content/pages/about.html

```diff
@@ -0,0 +1,396 @@
+                <span class="content">
+                  {{ header_2 }}
+                  <span class="sub header">Update your documentation whenever you push changes</span>
+                </span>
+              </a>
+
+              {%- set header_3 = "Preview your changes" %}
+              {%- set icon_3 = "fa-code-compare" -%}
+              <a href="#review" class="ui small header item">
+                <i class="fad {{ icon_3 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_3 }}
+                  <span class="sub header">Verify changes to your documentation during review</span>
+                </span>
+              </a>
+
+              {%- set header_4 = "Everything in one place" %}
+              {%- set icon_4 = "fa-box-check" -%}
+              <a href="#hosting" class="ui small header item">
+                <i class="fad {{ icon_4 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_4 }}
+                  <span class="sub header">We manage building, hosting, authentication, search, and analytics for you</span>
+                </span>
+              </a>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+  </div>
+</section>
+
+{% macro about_divide(text=None) %}
+  <div class="ui text container">
+    <div class="ui {%- if text %} horizontal {%- endif %} divider">
+      {% if text %}{{ text }}{% endif %}
+    </div>
+  </div>
+{% endmacro %}
+
+{{ about_divide() }}
+
+<section>
+  <div class="ui very padded container">
+    <div class="ui vertically padded centered grid" id="authoring">
+
+      {% call about.step(
+          header=header_1,
+          image="/images/about/authoring.png",
+          image_alt="Terminal window showing commands to commit documentation changes",
+          icon=icon_1) %}
+        {% markdown %}
+        Manage software documentation using all of the **same tools and workflow**
+        you're already using for development. Write docs with your favorite editor,
+        commit changes using version control, and preview changes during review.
+        {% endmarkdown %}
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Versioning", icon="fad fa-layer-group") %}
+        Give your users documentation that won't go out of date.
+        Build multiple versions from commits, branches, or tags in your repository.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Localization", icon="fad fa-language") %}
+        Integrate with localization tools to translate your documentation in multiple languages.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Track change history", icon="fad fa-code-commit") %}
+        Use the same version control system that you use for development to retain change history for your documentation.
+      {% endcall %}
```

Comment:
This isn't really a feature of our platform, another place I'd probably highlight benefits or our platform features instead.

Reply:
Aye, but this page is not intended to just describe features. It is describing how the service works from the user's perspective. I think it's important to describe vcs, as it's a fundamental working piece of our workflow.